Rarime/Code/Modules/Identity/Views/ZkLivenessIntroView.swift

```diff
@@ -0,0 +1,66 @@
+import SwiftUI
+
+struct ZkLivenessIntroView: View {
+    let onStart: () -> Void
+
+    var body: some View {
+        ZStack(alignment: .top) {
+            Circle()
+                .foregroundStyle(Gradients.gradientSixth)
+                .frame(width: 400, height: 394)
+                .offset(x: -0, y: -300)
```

Comment:
if the UI doesn't change, then try using the simple one: .offset(y: -300)
```suggestion
                .offset(y: -300)
```